### DISCUSSION TITLE
Pull Request for Wikis

### DISCUSSION THREAD

**Discussion by ShockedCoder** [community/community#9473] [Pull Requests]
The title

**Comment by michael-hawker** [comment]
We adopted what the VS Code team did and created a separate repo dedicated to a Wiki. Then when that gets a commit, it pushes that to the wiki repo in the other repo. That enables you to have issues and PRs and all that around your Wiki. It would be nice if that could just be an option to setup/connect more easily though.

**Comment by 201flaviosilva** [comment]
It’s not the best option, but I'm using this to try "fix" this issue: https://github.com/Andrew-Chen-Wang/github-wiki-action

**Comment by Acumane** [comment]
This has been a wish of users for the last decade, so I suspect GH Actions are the only solution we'll be getting.

[Bi-directional Wiki Sync](https://github.com/marketplace/actions/bi-directional-wiki-sync-action) gets you 99% of the way. imo it's the best choice, but check the [marketplace for others](https://github.com/marketplace?type=actions&query=wiki) in case that changes one day.
